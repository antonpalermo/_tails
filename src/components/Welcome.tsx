import React from "react"

export interface WelcomeProps {
  name: string
}

export default function Welcome({ name }: WelcomeProps) {
  return <h1>Welcome {name}</h1>
}
