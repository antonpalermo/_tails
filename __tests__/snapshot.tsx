import { render } from "@testing-library/react"

import Home from "@pages/index"
import Welcome from "@components/Welcome"

it("render home page unchanged", () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})

it("render welcome page unchanged", () => {
  const { container } = render(<Welcome name="Anton!" />)
  expect(container).toMatchSnapshot()
})
