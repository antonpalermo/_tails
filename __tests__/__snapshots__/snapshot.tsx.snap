// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render home page unchanged 1`] = `
<div>
  <h1>
    Welcome 
    User!
  </h1>
</div>
`;

exports[`render welcome page unchanged 1`] = `
<div>
  <h1>
    Welcome 
    Anton!
  </h1>
</div>
`;
